Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Pre 1.3.0 Fixes

## Changelog:
- Fixed errors mentioning missing:
  - Cultures
  - Culture groups
  - Faiths
  - Religions
  - Titles
  - Holy sites
- Replaced vanilla current_date by our triggers.
- Fixed https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/issues/249.

# Tasks:
Being on 1.2.2 CK3 version fix errors mentioned in `C:\Users\<User>\Documents\Paradox Interactive\Crusader Kings III\logs\error.log` mentioning missing...
- [x] Cultures
- [x] Culture groups, error name `event_target_links.cpp:1267`
- [x] Faiths, error name `event_target_links.cpp:1212`
- [x] Religions, error name `event_target_links.cpp:945`
- [x] Regions
- [x] Titles, error name `event_target_links.cpp:1832`
- [x] Holy sites, error mentions `controls_holy_site`

Once you take a task, mark it and assign yourself to the PR.


# How to test:
Check the [Files changed](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/246/files).

Open `error.log`, check for missing things mentioned in the changelog.
